crates/burrego/src/evaluator.rs

```diff
@@ -169,12 +169,18 @@ impl Evaluator {
             data = serde_json::to_string(&data)?.as_str(),
             "setting policy data"
         );
+        if let Some(deadline) = self.epoch_deadline {
+            self.store.set_epoch_deadline(deadline);
```

Comment:
A bit clunky to need to set the deadline for each wasm evaluation separately (in this case the json parse fun). I fear I would forget some at some point. We could pass down `Option<Deadline>` and call `store.set_epoch_deadline()` inside of `policy.set_data()`, `policy.evaluate()` etc.

Reply:
Yes, this got refactored by using a macro. Luckily this is something internal, that the end user doesn't have to mess with.

Can I mark this conversation as resolved?

Reply:
yep, totally, resolving!